assets/src/components/reporting/ReportingDashboard.tsx

```diff
@@ -133,6 +153,13 @@ class ReportingDashboard extends React.Component<Props, State> {
             </Box>
             <MessagesPerDayChart data={this.formatDailyStats()} />
           </Box>
+
+          <Box mb={4} mx={3} sx={{height: 320, flex: 1}}>
+            <Box mb={2}>
+              <Text strong>Messages per user</Text>
+            </Box>
+            <MessagesPerUserChart data={this.formatUserStats()} />
+          </Box>
           {/*
```

Comment:
let's remove the code on line 188 (https://github.com/papercups-io/papercups/pull/289/files#diff-2160a0e22b7bdfee0ff3c1561656c30fR188) now that this is here :)